dashboard/app/assets/javascripts/client_state.js

```diff
@@ -37,10 +37,27 @@ var COOKIE_OPTIONS = {expires: dashboard.clientState.EXPIRY_DAYS, path: '/'};
 dashboard.clientState.reset = function() {
   $.removeCookie('progress', {path: '/'});
   $.removeCookie('lines', {path: '/'});
+  $.removeCookie('puzzle_rating', {path: '/'});
```

Comment:
Could you do this with localStorage instead of cookies?  We tend to overflow the per-domain cookie limit sometimes.